File: app/src/main/java/com/xi/android/kantukuang/sinablog/TagResultUrl.java
package com.xi.android.kantukuang.sinablog;

import com.google.api.client.http.GenericUrl;
import com.google.api.client.util.Key;

public class TagResultUrl extends GenericUrl {
    @Key
    public String tag;
    @Key
    public int page;

    public TagResultUrl() {
        super("http://qing.blog.sina.com.cn/blog/api/tagresult.php");
    }
}

File: app/src/main/scala/com/shen/xi/android/tut/sinablog/TagResultUrl.scala
package com.shen.xi.android.tut.sinablog

import com.google.api.client.http.GenericUrl
import com.google.api.client.util.Key

class TagResultUrl extends GenericUrl("http://qing.blog.sina.com.cn/blog/api/tagresult.php") {
    @Key
    var tag: String = ""
    @Key
    var page: Int = 0
}
